diff --git a/build/index.js b/build/index.js
index 30f3b1d217c0d7ca7d5af2f8d4caa736bf997d36..62af185e27fdcf42b41b05ef0065189200ee562f 100644
--- a/build/index.js
+++ b/build/index.js
@@ -550,7 +550,7 @@ function wwwAuth(scheme, params) {
 }
 export function parseWwwAuthenticateChallenges(response) {
     if (!(response instanceof Response)) {
-        throw new TypeError('"response" must be an instance of Response');
+        // throw new TypeError('"response" must be an instance of Response');
     }
     if (!response.headers.has('www-authenticate')) {
         return undefined;
@@ -826,7 +826,7 @@ async function processGenericAccessTokenResponse(as, client, response, ignoreIdT
     assertAs(as);
     assertClient(client);
     if (!(response instanceof Response)) {
-        throw new TypeError('"response" must be an instance of Response');
+        // throw new TypeError('"response" must be an instance of Response');
     }
     if (response.status !== 200) {
         let err;